### ISSUE TITLE
are messages in model.fit(....) logged into some file?

### ISSUE THREAD

**Issue by yuconglin** [dmlc/mxnet#6222]
when I ran the model.fit function, it printed on screen:
INFO:root:Epoch[0] Batch [8]	Speed: 4.61 samples/sec	Train-accuracy=0.083333
INFO:root:Epoch[0] Batch [16]	Speed: 4.60 samples/sec	Train-accuracy=0.109375
INFO:root:Epoch[0] Batch [24]	Speed: 4.58 samples/sec	Train-accuracy=0.203125
INFO:root:Epoch[0] Batch [32]	Speed: 4.57 samples/sec	Train-accuracy=0.171875
.....................

instead of printing on screen, can such messages be saved to a log file?
Thanks

**Comment by patriciaaa82** [comment]
You can redirect your logging output to a file:

logging.getLogger().setLevel(logging.DEBUG)  # logging to stdout
logging.basicConfig(filename=filename,level=logging.DEBUG) #logging to a file
